Added a new page for Wwise Integration to UE4.27 for proper sound editing.

Kordusain opened the pull request:
UE's xaudio sound cues do not work in the gamepass version for some reason, so making a mod with custom audio that works on all platforms needs Wwise integration. Plus this gives us some flexibility in audio design, like making randomized sounds for a single event like the game does for picking up loose cash, setting up full tracks without needing to hex edit, and tying the audio levels to the ingame sliders.

I'm not sure if I should credit [Sanic](https://github.com/Sanic5238/) for figuring most of it out within the text or not.

Kordusain committed:
Proposed changes to Custom Music

Custom Music changed to Custom Audio
Added a sub-page for Wwise Integration

Kordusain committed:
Minor fix

Clarification on requirements.

Kordusain committed:
Minor cleanup

Prerequistes and some better intro.

Kordusain committed:
Added Legal disclaimer

Don't share the Wwise folder kids.

Kordusain committed:
More clarity and fixed slashes

Very minor stuff.

Kordusain committed:
Even more clarification

Wwise plugin share woes.

narknon commented on docs/custom-audio/integrating-wwise-to-unreal.md:
```diff
@@ -0,0 +1,189 @@
+
+Run the Moolah Template's PAYDAY3.uproj once, go to Editor Preferences > Source Code, set it to Visual Studio 2019. Close Unreal Editor.
+Go to your new empty Moolah Template folder, go to `MoolahProject-main\Plugins\Engine\Plugins\Wwise\Source\AkAudio\Public\` and grab `ESBZAkPoolType.h` and copy it somewhere safe for now.
+Delete the entire Wwise folder in `Plugins\Engine\Plugins`.
```
Wrong path

Kordusain replied:
Fixed.

narknon commented on docs/custom-audio/integrating-wwise-to-unreal.md:
```diff
@@ -0,0 +1,189 @@
+
+### Prerequisites
+
+- [Visual Studio 2019](https://visualstudio.microsoft.com/vs/older-downloads/)
```
You don't need 2019, 2017-2022 should work.

Kordusain replied:
Fixed, plus linked UE docs for needed submodules.

narknon commented on docs/custom-audio/integrating-wwise-to-unreal.md:
```diff
@@ -0,0 +1,189 @@
+
+
+        include
+        Win32_*
```
Don't need win32

Kordusain replied:
Wwise lied to me. Fixed.

narknon commented on docs/custom-audio/integrating-wwise-to-unreal.md:
```diff
@@ -0,0 +1,189 @@
+    "Microsoft.VisualStudio.Component.VC.ASAN",
+    "Microsoft.VisualStudio.Workload.NativeGame"
+  ]
+}
```
No idea what's going on here

Kordusain replied:
Removed, along with the image, because of clarification added to VS submodules.

CoolLKKPS commented:
add some pull request in ur branch: https://github.com/Kordusain/MoolahWikiWwiseIntegration/pull/1
just make things easier

Kordusain committed:
Complying with requested changes

Fixed file path, clarified Visual Studio versions and linked to UE docs for required submodules, got rid of the vsconfig codeblock. Cleared some more text.

Kordusain committed:
Delete vs19.png

Not needed anymore due to requested changes. My stupid ass forgor 💀

florensie committed:
First pass of fixes
